lib/aggregation/aggregator/src/test/fixtures/usageDocumentFieldsConstants.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const testCollectedUsageID = 'collector-id';
+const testResourceID = 'test-resource';
+const testOrganizationID = 'a3d7fe4d-3cb1-4cc3-a831-ffe98e20cf28';
+const testSpaceID = 'aaeae239-f3f8-483c-9dd0-de5d41c38b6a';
+const testConsumerID = 'external:bbeae239-f3f8-483c-9dd0-de6781c38bab';
+const testPlanID = 'basic';
+const testResourceType = 'test-resource-type';
+const testAccountID = '1234';
+const testMeteringPlanID = 'test-metering-plan';
+const testRatingPlanID = 'test-rating-plan';
+const testPricingPlanID = 'test-pricing-basic';
+
+const testResourceInstanceIDs = ['0b39fa70-a65f-4183-bae8-385633ca5c87', '1b39fa70-a65f-4183-bae8-385633ca5c88'];
+
+const times = {
+  endOfOctoberTwelveAM: 1446249600000, // Saturday, October 31, 2015 12:00:00 AM
```

Comment:
Any reasons not to use `moment(<date>)` here to get rid of the comments? Is it stubbed/spied in the tests?